src/SprykerEco/Zed/Computop/Business/Payment/Handler/Saver/Init/PayuCeeSingleResponseSaver.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Business\Payment\Handler\Saver\Init;
+
+use Generated\Shared\Transfer\ComputopPayuCeeSingleInitResponseTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+
+class PayuCeeSingleResponseSaver extends AbstractResponseSaver
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function save(QuoteTransfer $quoteTransfer): QuoteTransfer
+    {
+        $responseTransfer = $quoteTransfer->getPayment()->getComputopPayuCeeSingle()->getPayuCeeSingleInitResponse();
+        $this->setPaymentEntity($responseTransfer->getHeader()->getTransId());
+        if ($responseTransfer->getHeader()->getIsSuccess()) {
+            $this->getTransactionHandler()->handleTransaction(
+                function () use ($responseTransfer) {
+                    $this->savePaymentComputopEntity($responseTransfer);
+                    $this->savePaymentComputopDetailEntity($responseTransfer);
+                    $this->savePaymentComputopOrderItemsEntities();
+                }
+            );
+        }
+
+        return $quoteTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\ComputopPayuCeeSingleInitResponseTransfer $responseTransfer
+     *
+     * @return void
+     */
+    protected function savePaymentComputopEntity(ComputopPayuCeeSingleInitResponseTransfer $responseTransfer)
+    {
+        $paymentEntity = $this->getPaymentEntity();
+        $paymentEntity->setPayId($responseTransfer->getHeader()->getPayId());
+        $paymentEntity->setXId($responseTransfer->getHeader()->getXId());
+        $paymentEntity->save();
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\ComputopPayuCeeSingleInitResponseTransfer $responseTransfer
+     *
+     * @return void
+     */
+    protected function savePaymentComputopDetailEntity(ComputopPayuCeeSingleInitResponseTransfer $responseTransfer)
```

Comment:
return type

Reply:
Moved to entity manager and added return type